src/main/java/org/fairdatatrain/trainhandler/api/controller/StationDirectoryController.java

```diff
@@ -78,4 +78,10 @@ public StationDirectoryDTO update(
     public void delete(@PathVariable UUID uuid) throws NotFoundException {
         stationDirectoryService.delete(uuid);
     }
+
+    @DeleteMapping(path = "/{uuid}/stations")
+    @ResponseStatus(code = HttpStatus.NO_CONTENT)
+    public void refreshStations(@PathVariable UUID uuid) throws NotFoundException {
+        stationDirectoryService.reindex(uuid);
+    }
```

Comment:
http `DELETE`? Not sure what the alternative would be, `PUT` with a `?refresh` parameter?